src/schema/v2/image/index.ts

```diff
@@ -58,7 +58,7 @@ const ImageType = new GraphQLObjectType<any, ResolverContext>({
     },
     isDefault: {
       type: GraphQLBoolean,
-      resolve: ({ is_default }) => is_default,
+      resolve: ({ default: is_default }) => is_default,
```

Comment:
Looks like there is in fact an image type in gravity that has an `is_default` property. It is called `AdditionalImage` . Could they both be passed through this resolver in different contexts? That would explain all the existing code and fixtures using the `is_default` property name. If so, maybe this should be

 ```suggestion
      resolve: (image) => image.is_default || image.default,
```

and then you can update the response typings to include both as optional properties?

Reply:
Yep, agree w/ @ds300 above